lib/modules.nix

```diff
@@ -710,6 +710,7 @@ rec {
 
   mkOptionDefault = mkOverride 1500; # priority of option defaults
   mkDefault = mkOverride 1000; # used in config sections of non-user modules to set a default
+  mkImageMediaOverride = mkOverride 60; # image media profiles can be derived by inclusion into host config, hence needing to override host config, but do allow user to mkForce
```

Comment:
You'd have to inherit this in `lib/default.nix` for this to work. This way it blocks unstable eval

Reply:
oh noes